src/analyzer/protocol/krb/KRB.h

```diff
@@ -2,15 +2,15 @@
 
 #pragma once
 
+// This is needed for USE_KRB5 below.
```

Comment:
Ignore if you don't feel like, but I guess we could actually add a `-DUSE_KRB5` compile flag for the KRB source files, too?